todo-app/src/components/Todos/Todo-Avatar.js

```diff
@@ -0,0 +1,11 @@
+// jshint esnext: true
+
+import React from 'react';
+
+export default class Avatar extends React.Component {
+    render(){
+        return (
+            <img src={this.props.avatar}/>
```

Comment:
If you run your app with `yarn start`, you'll see this warning in the console:

    ./src/components/Todos/Todo-Avatar.js
      Line 8:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

This is an accessibility thing, you can check out [docs here](https://www.w3schools.com/tags/att_img_alt.asp).